pkg/proxy/client/client_conn.go

```diff
@@ -72,8 +72,8 @@ func (cc *ClientConnection) Run(ctx context.Context) {
 	}
 
 clean:
-	// graceful close
-	if errors.Is(err, os.ErrDeadlineExceeded) {
+	// graceful close or force close when shutting down
+	if errors.Is(err, os.ErrDeadlineExceeded) || errors.Is(err, net.ErrClosed) {
```

Comment:
I think we can ignore errors with `!clientErr && (errors.Is(err, os.ErrDeadlineExceeded) || errors.Is(err, net.ErrClosed))` for logging, instead of direct return.

Reply:
Do you mean still log them? They will be too many.
BTW, clientErr is true for these cases.

Reply:
> Do you mean still log them? They will be too many.

No I meant to ignore them. But we should judge `clientErr`, too, since it may come from server somehow, too.

>  BTW, clientErr is true for these cases.

Oh, yes, I wrote it wrong.  Maybe:

```
if clientErr &&
 (errors.Is(err, io.EOF) || errors.Is(err, os.ErrDeadlineExceeded) || errors.Is(err, net.ErrClosed)) {
   return
}
cc.logger.Info(...)
```